data-prepper-plugins/s3-source/src/test/java/org/opensearch/dataprepper/plugins/source/configuration/SnappyCompressionOptionTest.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.configuration;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.EnumSource;
+import org.opensearch.dataprepper.plugins.source.compression.CompressionEngine;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class SnappyCompressionOptionTest {
+    @ParameterizedTest
+    @EnumSource(CompressionOption.class)
+    @CsvSource({"SNAPPY"})
+    void fromOptionValue(final CompressionOption option) {
```

Comment:
I think you can remove this test entirely. It is already covered in `CompressionOptionTest`.

https://github.com/opensearch-project/data-prepper/blob/5110699c993d4e17a56ae7d1f8c11def0429c0d9/data-prepper-plugins/s3-source/src/test/java/org/opensearch/dataprepper/plugins/source/configuration/CompressionOptionTest.java#L17-L22

Reply:
Removed it